detekt-rules-complexity/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/complexity/RedundantMap.kt

```diff
@@ -0,0 +1,43 @@
+package io.gitlab.arturbosch.detekt.rules.complexity
+
+import io.gitlab.arturbosch.detekt.api.CodeSmell
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.Debt
+import io.gitlab.arturbosch.detekt.api.Entity
+import io.gitlab.arturbosch.detekt.api.Issue
+import io.gitlab.arturbosch.detekt.api.Rule
+import io.gitlab.arturbosch.detekt.api.Severity
+import io.gitlab.arturbosch.detekt.rules.IT_LITERAL
+import io.gitlab.arturbosch.detekt.rules.MAP_LITERAL
+import org.jetbrains.kotlin.psi.KtLambdaExpression
+
+/**
+ * map { it } does not make sense. You can remove the operator without any impact.
```

Comment:
```suggestion
 * Using map { it } indicates a remnant of a recent refactoring. You can remove the operator without any impact.
```

_does not make sense_ is really strong in this context.